example/lib/screens/native_ads.dart

```diff
@@ -43,32 +67,32 @@ class _NativeAdsState extends State<NativeAds>
       child: ListView(
         padding: EdgeInsets.all(10),
         children: [
-          if (controller.isLoaded)
-            NativeAd(
-              controller: controller,
-              height: 60,
-              builder: (context, child) {
-                return Material(
-                  elevation: 8,
-                  child: child,
-                );
-              },
-              buildLayout: adBannerLayoutBuilder,
-              loading: Text('loading'),
-              error: Text('error'),
-              icon: AdImageView(padding: EdgeInsets.only(left: 6)),
-              headline: AdTextView(style: TextStyle(color: Colors.black)),
-              advertiser: AdTextView(style: TextStyle(color: Colors.black)),
-              body: AdTextView(
-                  style: TextStyle(color: Colors.black), maxLines: 1),
-              media: AdMediaView(height: 70, width: 120),
-              button: AdButtonView(
-                margin: EdgeInsets.only(left: 6, right: 6),
-                textStyle: TextStyle(color: Colors.green, fontSize: 14),
-                elevation: 18,
-                elevationColor: Colors.amber,
-              ),
+          //if (controller.isLoaded)
```

Comment:
This part of the example app shouldn't be changed.

Reply:
Since the first ad in the example did not have some of the details (like Image) I just attached the controller to the last ad so that I could showcase the accessibility of all of the details. However I can revert this change if you think it'd be better to attach it to the first one.

Reply:
Similar to what I did here

https://github.com/bdlukaa/native_admob_flutter/blob/master/lib/src/native/controller/controller.dart#L127 

and here

https://github.com/bdlukaa/native_admob_flutter/blob/master/lib/src/native/controller/controller.dart#L300

When the ad is loaded, the platform side sends more info about the native ad.